### DISCUSSION TITLE
Can we have some basic documentations for this refactored ALS?

### DISCUSSION THREAD

**Discussion by Bigbossbro08** [Sixze/ALS-Refactored#102] [Ideas]
This project is very different than ALS. So, what goes where is extremely difficult to understand without much guidelines. Like adding Overlay States and what will go where. How GetParentUnsafe and other functions interact with each other. How values are being moved around between linked layers? Would be nice various of it be documented.

**Comment by Sixze** [comment]
Unfortunately, I don't have much free time to write full documentation, but this plugin is still ALS, and if you've had experience with other ALS versions, then it shouldn't be too hard for you to understand what's going on inside. You can always ask a question about specific details of my ALS version and I will answer them whenever possible.

**Reply by Bigbossbro08** [reply]
I don't think you need think you need all the nitty-gritty details of full documentation. Let's say we need to know "How to add an Overlay State?" in a doc folder. Then layout the basics for it. Later, using pull request and such we can expand all the documentations we need. Start as short later can be expanded. Its not a critical code change to worry about.
